AirCasting/CoreData/UIStorage.swift

```diff
@@ -0,0 +1,69 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import CoreData
+import Combine
+import SwiftUI
+import Resolver
+
+protocol UIStorage {
+    func accessStorage(_ task: @escaping(HiddenCoreDataUIStorage) -> Void)
+}
+
+protocol UIStorageContextUpdate {
+    func cardStateToggle(for sessionUUID: SessionUUID) throws
+    func changeStream(for sessionUUID: SessionUUID, stream: String) throws
+    func save() throws
+}
+
+final class CoreDataUIStorage: UIStorage {
+    
+    @Injected private var persistenceController: PersistenceController
+    private lazy var updateSessionParamsService = UpdateSessionParamsService()
```

Comment:
would be nice to move `UpdateSessionParamsService` to `Resolver` (be a nice scout ;) )